packages/jobs/lib/app.ts

```diff
@@ -63,6 +66,16 @@ try {
         // not closing on purpose
     });
 
+    if (envs.RUNNER_TYPE !== 'REMOTE') {
+        if (isLocal) {
+            // when running locally, the runners (running as processes) are being killed
+            // when the main process is killed and the fleet entries are therefore not associated with any running process
+            // we then must fake a new deployment so fleet replaces runners with new ones
+            await runnersFleet.deploy(generateCommitHash());
```

Comment:
it's handy but will create a lot of rows, out of curiosity why not strip the `deployments` and just use current hash?

Reply:
because each local execution (even from the same current hash) is in reality a different runner so it has to be different for each execution.
wrt to the number of rows, even if you run nango locally hundreds of times a day, it will never reach any big number that pg cannot handle easily